pyproject.toml

```
@@ -42,6 +42,7 @@ numpy = [
   { version = ">=2.1", python = "^3.13" }
 ]
 hightime = { version = ">=0.2.2", allow-prereleases = true }
+typing_extensions = { version = ">=4.0.0" }
```

Restore the previous dependency
```suggestion
typing-extensions = ">=4.13.2"
```